src/events/gamebanana_watcher.js

```diff
@@ -0,0 +1,144 @@
+                                    'https://gamebanana.com/mods/',
+                                    '',
+                                );
+                                let mapPreviewImage = `https://gamebanana.com/mods/embeddables/${modID}?type=large`;
+                                let mapName = res[0]['_sName'];
+                                let mapAuthor = res[0]['_aOwner']['_sUsername'];
+                                let mapDL = `https://gamebanana.com/mods/${modID}`;
+                                let updated = false;
+
+                                if (csgoMaps.find((x) => x.name == mapName)) {
+                                    // If the map has not changed since last time we checked it.
+                                    if (
+                                        res[0]['_tsDateUpdated'] ==
+                                        csgoMaps.find((x) => x.name == mapName)
+                                            .dateLastUpdated
+                                    )
+                                        return;
+                                    else updated = true;
+                                }
+
+                                // Add the new map object to the array to prevent duplicate messages.
+                                csgoMaps.push({
+                                    name: mapName,
+                                    dateAdded: res[0]['_tsDateAdded'],
+                                    dateLastUpdated: res[0]['_tsDateUpdated'],
+                                });
+
+                                const embed = new MessageEmbed()
+                                    .setTitle(
+                                        updated
+                                            ? 'CS:GO - Map Updated'
+                                            : 'CS:GO - New Map',
+                                    )
+                                    .setColor(updated ? 'YELLOW' : '#00D166')
+                                    .addFields([
+                                        {
+                                            name: 'Map',
+                                            value: `${mapName} `,
+                                        },
+                                        {
+                                            name: 'Creator',
+                                            value: `${mapAuthor} `,
+                                        },
+                                        {
+                                            name: 'Download',
+                                            value: `${mapDL} `,
+                                        },
+                                    ])
+                                    .setImage(mapPreviewImage);
+                                channel.send({
+                                    embeds: [embed],
+                                });
+                            });
```

Comment:
This part is pretty much identical to the code for checking CSS submissions and it can probably be made into its own function that handles both cases.